pkg/openshift/changeset.go

```diff
@@ -27,6 +27,7 @@ var (
 		"Deployment":            "m",
 		"Service":               "n",
 		"Route":                 "o",
+		"Job":                   "p",
```

Comment:
This is used for ordering, so I think it is best to have this sorted next to `CronJob`, e.g. as `j`, and all below ones move down one letter.